Pyboard.py

```diff
@@ -0,0 +1,19 @@
+from Microcontroller import Microcontroller
+import pyb.Pin.board as pins
+import pyb
+
+class Pyboard(Microcontroller):
+
```

Comment:
please overload `__init__` as follows

```python
def __init__(self):
    self.set_RGB_data()
    self.set_row_select()
```

in this way, the defaults are automatically applied at init time

alternatively, you could add those method calls to the parent init